src/main/java/JavaLessons/Lesson1/Lesson3.java

```diff
@@ -0,0 +1,222 @@
+package JavaLessons.Lesson1;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Lesson3 {
+    public static void main(String[] args) {
+        square();
+        zeroToOneAndViceVersa();
+        array100();
+        multiplyUnderSix();
+        multiplyUnderSixV2();
+        crossArray();
+        lenArray(6, 15);
+        minMaxArray(10);
+        minMaxArrayV2(4);
+        System.out.println(sumTwoPartArray(3, 4, 5, 7, 2, 2, 1));
+        System.out.println(sumTwoPartArrayV2(3, 4, 5, 7, 2, 2, 1));
+        shiftArray(5, 5, 7, 6, 6);
+        shiftArrayV2(3, 5, 7, 6, 2);
+        shiftArrayV3(5, 10, 34, 5, 12, 6, 3, 5);
+    }
+
+    public static void square() {
+        for (int i = 0; i < 6; i++) {
+            for (int j = 0; j < 6; j++) {
+                if ((i == 2 || i == 3) && (j == 2 || j == 3)) {
+                    System.out.print("  ");
+                } else {
+                    System.out.print("* ");
+                }
+            }
+            System.out.println();
+        }
+    }
+
+    // Задать целочисленный массив, состоящий из элементов 0 и 1. Например: [ 1, 1, 0, 0, 1, 0, 1, 1, 0, 0 ]. С помощью цикла и условия заменить 0 на 1, 1 на 0;
+    public static void zeroToOneAndViceVersa() {
+        System.out.print("int[] arr = {");
+        int[] arr = {1, 1, 0, 0, 1, 0, 1, 1, 0, 0};
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = (arr[i] == 0) ? 1 : 0;
+            System.out.print(arr[i]);
+            if (i < arr.length -1) {
+                System.out.print(", ");
+            }
+        }
+        System.out.print("}\n");
+    }
+
+// Задать пустой целочисленный массив длиной 100. С помощью цикла заполнить его значениями 1 2 3 4 5 6 7 8 … 100;
+    public static void array100() {
+        System.out.print("int[] arr = {");
+        int[] arr = new int[100];
+        for (int i = 0; i < arr.length; i++) {
+           arr[i] = i + 1;
+           System.out.print(arr[i]);
+            if (i < arr.length -1) {
+                System.out.print(", ");
+            }
+        }
```

Comment:
This code is correct, however you should take into account a couple of nuances:
1. It seems that you use the same piece of code in a few methods -> that means you should extract it into another method and just call it. There is such principle like DRY, don't repeat yourself. *It's not a dogma, there might be some cases where copypaste might be a better solution, but they are marginal. 
2. There is another general Single responsibility principle. Currently it might be better to split filling functionality and printing. (there might be a lot of exceptions) But in general everything is ok.